configure

```diff
@@ -1727,7 +1727,8 @@ Some influential environment variables:
               Specifies whether the package will run in 32- or 64-bit mode.
               One of: 31,32,64
   OMR_TOOLCHAIN
-              The toolchain used to build the package. One of: gcc,xlc,msvc
+              The toolchain used to build the package. One of:
+              gcc,xlc,msvc,openxl
```

Comment:
This file should not be modified: it's not used in cmake builds.

Reply:
removed